### PR TITLE
Fixes /mob/dead/new_player harddels

### PR DESCRIPTION
**Pull request by Dejaku51**
## About The Pull Request
Ports https://github.com/tgstation/tgstation/pull/59044
Ports https://github.com/tgstation/tgstation/pull/60072
Partly ports https://github.com/tgstation/tgstation/pull/63877 (tgui fixes)

## Why It's Good For The Game
/mob/dead/new_player harddels is second most harddeleting thing (see #dev-announcements).
This PR should fix that

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl: Dejaku51, Mothblocks, LemonInTheDark
fix: Fixes /mob/dead/new_player harddels
/:cl:

### CODE REVIEW

File: code/game/gamemodes/dynamic/dynamic_simulations.dm [hunk 1]
```diff
@@ -119,19 +119,5 @@
 
 	return export
 
-/client/proc/export_dynamic_json()
```
Comment (Penwin0) [inline code/game/gamemodes/dynamic/dynamic_simulations.dm:122]: Is this being removed because it's no longer functional?

Comment (Dejaku51) [reply code/game/gamemodes/dynamic/dynamic_simulations.dm:122]: i think so

Comment (Dejaku51) [reply code/game/gamemodes/dynamic/dynamic_simulations.dm:122]: roundstart_rules won't work for sure

**Review by PowerfulBacon — APPROVED** [review]

File: code/modules/tgui/tgui.dm [hunk 2]
```diff
@@ -258,7 +258,7 @@
 		return
 	var/datum/host = src_object.ui_host(user)
 	// If the object or user died (or something else), abort.
-	if(!src_object || !host || !user || !window)
+	if(!QDELETED(src_object) || !QDELETED(host) || !QDELETED(user) || !QDELETED(window))
```
Comment (PowerfulBacon) [inline code/modules/tgui/tgui.dm:261]: Fuck

**Pull request merged**